File: 344-reverse-string/344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        
        size  = len(s)
        
        low = 0
        high = size-1
        while(low < high):
            
            s[low] , s[high] = s[high] , s[low]
            
            low +=1
            high -=1

File: 344-reverse-string/344-reverse-string.rs
impl Solution {
    pub fn reverse_string(s: &mut Vec<char>) {
     
        let mut start = 0;
        let mut end = s.len()-1;
        
        while start <end{
            let c1 = s[start];
            let c2 = s[end];
            
            s[start] = c2;
            s[end] = c1;
            
            start +=1;
            end -=1;
        }
    }
}